.github/workflows/bpf-checks.yaml

```diff
@@ -16,7 +16,7 @@ jobs:
         with:
           fetch-depth: 0
       - name: Run checkpatch.pl
-        uses: docker://cilium/cilium-checkpatch:e6a6843cef50a43c38819badb55a21061967ce64
+        uses: quay.io/cilium/cilium-checkpatch:cc7e6b5811f46d7b040dedfe2f6b0010c2c51a12@sha256:687bd6c550449d2eaa0b258c19703a91093243b9138f8c551563a34df31179f0
```

Comment:
did you try `docker://quay.io/cilium/cilium-checkpatch:cc7e6b5811f46d7b040dedfe2f6b0010c2c51a12@sha256:687bd6c550449d2eaa0b258c19703a91093243b9138f8c551563a34df31179f0`?

Reply:
I suspect you gave me the wrong sha :). Works better with `sha256:9160b6ca58eb99a3ed5d567a494b2e2001325ebad32029c5bd17a8ae4df01044`.